src/Commands/ConstructCommand.php

```diff
@@ -152,22 +170,32 @@ protected function execute(InputInterface $input, OutputInterface $output)
         $testFramework = $this->testFrameworkWarning($testFramework, $output);
         $phpVersion = $this->phpVersionWarning($phpVersion, $output);
 
-        $this->settings = new Settings(
-            $projectName,
-            $testFramework,
-            $license,
-            $namespace,
-            $git,
-            $phpcs,
-            $keywords,
-            $vagrant,
-            $editorConfig,
-            $phpVersion,
-            $environment,
-            $lgtm,
-            $githubTemplates,
-            $codeOfConduct
-        );
+        if ($this->isConfigurationApplicable($configuration)
+            && $ignoreDefaultConfiguration === false) {
```

Comment:
This is a little hard to read. Could the `$ignoreDefaultConfiguration` variable be sent into the `isConfigurationApplicable()` function and have it checked there? 

Also, what happens if they run with `-c /mystuff/config.yml -i` It looks like that would skip their config entirely?